new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWITValidation.java

```diff
@@ -3,24 +3,45 @@
 
 package oracle.weblogic.kubernetes;
 
+import java.util.Arrays;
+import java.util.List;
+
 import oracle.weblogic.kubernetes.actions.TestActions;
+import oracle.weblogic.kubernetes.assertions.TestAssertions;
 import oracle.weblogic.kubernetes.extensions.LoggedTest;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import static oracle.weblogic.kubernetes.actions.ActionConstants.MODEL_DIR;
 import static oracle.weblogic.kubernetes.actions.TestActions.withWITParams;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @DisplayName("Simple validation of basic WIT functions")
 class ItWITValidation implements LoggedTest {
+  private static final String WDT_MODEL_FILE = "model1-wls.yaml";
+  private static final String IMAGE_NAME = "test-mii-image-2";
+  private static final String IMAGE_TAG = "v1";
+  
   @Test
   @DisplayName("Create a MII image")
   public void testCreatingMIIImage() {
 
-    // create the MII image
-    // TODO add model files and other contents to the image once we have those resources
-    boolean success = TestActions.createMIIImage(withWITParams());
+    logger.info("WDT model directory is " + MODEL_DIR);
+
+    // build the model file list
+    List<String> modelList = Arrays.asList(MODEL_DIR + "/" + WDT_MODEL_FILE);
 
+    // build an image using WebLogic Image Tool
+    boolean success = TestActions.createMIIImage(
+        withWITParams()
+            .modelImageName(IMAGE_NAME)
+            .modelImageTag(IMAGE_TAG)
+            .modelFiles(modelList)
+            .wdtVersion("latest")
+            .redirect(false));
+ 
     assertEquals(true, success, "Failed to create the image using WebLogic Deploy Tool");
+    TestAssertions.imageExists(IMAGE_NAME, IMAGE_TAG);
```

Comment:
pleas static import so you don't need `TestAssertions.` here

Reply:
`imageExists()` could probably use a better name too - maybe `assertImageExists`

Reply:
Made the change to use static import and rename the method.